services/app-api/src/postgres/contractAndRates/findAllRatesStripped.test.ts

```diff
@@ -113,7 +81,7 @@ it('returns all rates with stripped down data', async () => {
 
     await withdrawTestRate(cmsServer, rateBID, 'Withdraw invalid rate')
 
-    const strippedRatesOrErrors = must(await findAllRatesStripped(client))
+    const strippedRatesOrErrors = must(await findAllRatesStripped(client, {}))
```

Comment:
Nit: I avoid `{}`  for representing  empty/no args since it is not explicit. Prefer making the entire type of args optional.  This could be fixed later though